db/seeds/01_alta.rb

```diff
@@ -228,7 +228,7 @@
   end
 
   path = Rails.root.join("app", "assets", "images", "hero.jpg")
-  10.times do
+  50.times do
```

Comment:
Could we do the same for the baja seed file, also? Doesn't have to be 50, but more than 10 so pagination shows. Thanks!

Reply:
done!

Reply:
Thank you!